client/packages/programs/src/JsonForms/components/QuantityPrescribed.tsx

```diff
@@ -99,6 +101,19 @@ export const quantityPrescribedTester = rankWith(
   uiTypeIs('QuantityPrescribed')
 );
 
+const getEndOfSupply = (
+  baseTime: string,
+  pillCount: number,
+  options: Options | undefined
+): Date => {
+  return DateUtils.startOfDay(
+    DateUtils.addDays(
+      new Date(baseTime),
+      pillCount * (options?.quantityPerDay ?? 1)
+    )
+  );
+};
+
```

Comment:
I don't suppose we should also factor in the "Remaining Pill Count"?

Reply:
The naming is misleading QuantityPrescribed is the total quantity prescribed including the remaining pill count... I changed the label in the ui schema to make this clearer (next PR) will also check in today's meeting.

Sorry for having all these issues fixed in later branches but didn't noticed them earlier...